src/lib.rs

```diff
@@ -42,5 +41,4 @@ mod prototype;
 mod tree;
 mod linklabel;
 
-pub use crate::prototype::Parser;
-pub use crate::parse::{Alignment, Event, Tag, Options};
```

Comment:
I'm not sure how much of a problem deleting the `parse` mod creates for merging with master, maybe it would be good to experiment. But in the not too distant future (maybe as a followup PR after the merge), it makes sense to rename `prototype` to `parse`.